plugins/package-managers/gradle-plugin/src/main/kotlin/GradleModelExtensions.kt

```diff
@@ -44,7 +44,10 @@ internal fun Configuration.isRelevant(): Boolean {
     val isDeprecatedConfiguration = GradleVersion.current() >= GradleVersion.version("6.0")
             && this is DeprecatableConfiguration && resolutionAlternatives != null
 
-    return canBeResolved && !isDeprecatedConfiguration
+    // Do not try to resolve dependencies metadata configurations as there often cause failures with Gradle itself.
+    val isDependenciesMetadata = name.endsWith("DependenciesMetadata")
```

Comment:
Does this string always have a prefix or could the full config name also be "DependenciesMetadata" ? (lower first char)

Reply:
There is always a prefix.